JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +33,60 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        /**
+         * Authorized user id
+         */
+        private authId: string;
```

Comment:
I like the idea of hiding internal implementation details behind proper abstractions. In this particular case, I am concerned that 
- `private` visibility is strictly a TypeScript concept. Our API is predominantly consumed from JavaScript, where the `authId` property will appear as a regular public property. A common workaround for "hide" private members in JavaScript is a naming convention, such as starting internal member names with an underscore.
- We are introducing an inconsistency into the previously established API, making it more confusing than necessary.